identity-iota/src/tangle/client_map.rs

```diff
@@ -23,19 +23,26 @@ type State = DashMap<NetworkName, Arc<Client>>;
 #[derive(Debug)]
 pub struct ClientMap {
   data: State,
+  compression: bool,
 }
```

Comment:
1. `compression` should be set on the `Client` instead of the `ClientMap` to allow different settings per private Tangle, for example.
2. `compression` could be a `MessageEncoding` enum instead for forwards compatibility with any new encodings that are added.